### PR TITLE
Add keyboard shortcuts to search field

### PR DESCRIPTION
**Pull request by jevenski**
Made a few changes to the search field, which I feel may come in handy.

1. Keyboard shortcut "Ctrl+F" to focus on the search field, and "Esc" once to clear the search result, and "Esc" again to escape from searching.

2. A help note to the help dialog.
help-dialog

3. Category dropdown list hidden when searching locally, instead of disabling it.
search-field-before

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -291,6 +291,7 @@ var theUILang =
  Toggle_details			: "Toggle details",
  Toggle_categories		: "Toggle categories",
  Delete_current_torrents	: "Delete current torrent(s)",
+ Quick_search   : "Quick search",
```
Comment (stickz) [inline lang/en.js:294]: Could you fix the spacing in the language files, so it matches the other items?

File: plugins/extsearch/init.js [hunk 2]
```diff
@@ -735,7 +735,7 @@ plugin.refreshCategories = function()
 		for( var i=0; i<theSearchEngines.sites[theSearchEngines.current].cats.length; i++)
 			$('#exscategory').append("<option value='"+theSearchEngines.sites[theSearchEngines.current].cats[i]+"'>"+theSearchEngines.sites[theSearchEngines.current].cats[i]+"</option>");
 	}
-	$("#exscategory").prop("disabled",(theSearchEngines.current == -1));
+  $("#exscategory").prop("hidden", (theSearchEngines.current === -1));
```
Comment (stickz) [inline plugins/extsearch/init.js:738]: Formatting.

**Review by stickz — CHANGES_REQUESTED** [review]
Looks good, just need to adjust the formatting (spacing) to match the other items in the JavaScript files. You can resolve reviews on when you're done :)

File: lang/en.js (hunk 1 above)
Comment (jevenski) [reply lang/en.js:294]: Yes, I'd be glad to do so. Just want to make sure how do I do it properly. Do I fix the spacing and summit another commit, and the changes will be automatically added to this pull request?

**Comment by jevenski** [comment]
Submitted another commit regarding the indent format changes to this pull request. Everything looks better now.

**Pull request merged**